src/daft-scan/src/python.rs

```diff
@@ -319,6 +319,35 @@ pub mod pylib {
             Ok(Some(PyScanTask(scan_task.into())))
         }
 
+        #[staticmethod]
+        pub fn sql_scan_task(
+            url: String,
+            file_format: PyFileFormatConfig,
+            schema: PySchema,
+            storage_config: PyStorageConfig,
+            num_rows: Option<i64>,
+            size_bytes: Option<u64>,
+            pushdowns: Option<PyPushdowns>,
+        ) -> PyResult<Self> {
+            let data_source = DataFileSource::DatabaseDataSource {
+                path: url,
+                chunk_spec: None,
+                size_bytes,
+                metadata: num_rows.map(|n| TableMetadata { length: n as usize }),
+                partition_spec: None,
+                statistics: None,
```

Comment:
if the table is partitioned by some column, we could leverage that for statistics

Reply:
Will make an issue for this to do as a follow on.